Sources/XcodeGraph/Models/ProjectOptions.swift

```diff
@@ -59,6 +59,8 @@ extension Project.Options {
             testLanguage: String? = nil,
             testRegion: String? = nil,
             testScreenCaptureFormat: ScreenCaptureFormat? = nil,
+            captureScreenshotsAutomatically: Bool,
+            deleteScreenshotsWhenEachTestSucceeds: Bool,
```

Comment:
This technically constitutes a breaking change which we are trying to avoid.

I'd consider adding the new options to `TestingOptions` instead.

Reply:
I'd suggest to model these options more closer to how they are modeled in XcodeProj: https://github.com/tuist/XcodeProj/blob/9f26d78d72ef40dd2e35f624bbcde1e3b28762cf/Sources/XcodeProj/Scheme/XCScheme%2BTestAction.swift#L42

Reply:
> I'd consider adding the new options to TestingOptions instead.
got it. Will make the change right away.

Reply:
> I'd suggest to model these options more closer to how they are modeled in XcodeProj:
https://github.com/tuist/XcodeProj/blob/9f26d78d72ef40dd2e35f624bbcde1e3b28762cf/Sources/XcodeProj/Scheme/XCScheme%2BTestAction.swift#L42

What do you think about this [comment](https://github.com/yonaskolb/XcodeGen/pull/942#discussion_r484088072)? I believe that structuring variables closer to the xcodeproj model as you suggested might make it less user-friendly from an user perspective.